action.yml

```diff
@@ -5,42 +5,33 @@ branding:
   icon: 'monitor'
   color: 'gray-dark'
 inputs:
-  report_dir:
+  prev_git_hash:

```

Comment:
Thinking about this more, it might make sense to use a "list" of previous Git hashes instead just the single previous hash. The reason being is that not all commits might have test reports/history associated with them. One option is to wait and see if this change will be necessary and do it under another card if so.